docs/source/release_notes.rst

```diff
@@ -19,9 +19,10 @@ Future Release
     * Testing Changes
         * Update test cases to cover __main__.py file (:pr:`1927`)
         * Pin ``responses`` version to be below 0.19.0 in minimum_test_requirements to avoid moto error (:pr:`1929`)
+        * Add CI workflow to test with latest woodwork main branch (:pr:`1936`)
 
     Thanks to the following people for contributing to this release:
-    :user:`tamargrey`, :user:`kushal-gopal`, :user:`rwedge`, :user:`mingdavidqi`, :user:`andriyor`
+    :user:`tamargrey`, :user:`kushal-gopal`, :user:`rwedge`, :user:`mingdavidqi`, :user:`andriyor`, :user:`mingdavidqi`
```

Comment:
Your username is already in the list - don't need to add it again.

Reply:
updated